iFixit/react-commerce
# Footer Package: don't rely on @ifixit/ui

We are running into a ton of rush errors in https://github.com/iFixit/ifixit/pull/46933 where rush doesn't have an updated rush.json with all of the packages from @ifixit/ui. Instead of adding a ton of dependencies to this, lets just extract Menu from
@ifixit/ui into its own package and not rely on @ifixit/ui at all in the footer package.

## QA
Make sure the footer looks and functions the same as before

Connects to https://github.com/iFixit/ifixit/pull/46933 and https://github.com/iFixit/ifixit/issues/45964
